open_spiel/python/mfg/algorithms/distribution.py

```diff
@@ -132,9 +132,11 @@ def value(self, mfg_state):
     return self.value_str(
         mfg_state.observation_string(pyspiel.PlayerId.DEFAULT_PLAYER_ID))
 
-  def value_str(self, mfg_state_str):
-    try:
-      return self.distribution[mfg_state_str]
-    except KeyError as e:
-      raise ValueError(
-        f'Distribution not computed for state {mfg_state_str}') from e
+  def value_str(self, mfg_state_str, default_value = None):
+    if default_value is None:
```

Comment:
Can you add a docstring, saying that an exception will be raised if the value is not found and default_value is None?

Reply:
Done